srcpkgs/rinstall/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'rinstall'
+pkgname=rinstall
+version=0.2.0
+revision=1
+build_style=cargo
+short_desc="Declarative install for programs"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/danyspin97/rinstall"
+changelog="https://raw.githubusercontent.com/danyspin97/rinstall/main/CHANGELOG.md"
+distfiles="https://github.com/danyspin97/rinstall/archive/refs/tags/${version}.tar.gz"
+checksum=386ae4d52ad62b03304596d5dfc7bb273492ac3fed54f1110d0da8dc1847e4c6
```

Comment:
I think in this case, rset and rinstall should conflict

```suggestion
checksum=386ae4d52ad62b03304596d5dfc7bb273492ac3fed54f1110d0da8dc1847e4c6
conflicts="rset"
```

![let them fight](https://user-images.githubusercontent.com/5366828/227813295-fc54fc5f-2c85-48cc-92e9-ec68e758d7ac.gif)

Reply:
Done, thanks :)